android-build/Dockerfile

```diff
@@ -43,10 +43,14 @@ RUN echo y | android update sdk --no-ui --all --filter platform-tools | grep 'pa
 
 # --- Install android SDK
 # * If you are installing multiple SDK packages keep these in descending order!
+RUN echo y | android update sdk --no-ui --all --filter android-25 | grep 'package installed'
```

Comment:
We had a conversation internally and outcome of that was that we are indeed going to split out the Android image into separate ones for each SDK version. Also we identified that it might be better to have an image that is using the `alpine` image as it's base.

Another critical and quite large component is the image is the NDK. Not all projects need it. I will also split out the NDK version out of the main images.